validator/keymanager/remote-web3signer/client.go

```diff
@@ -2,38 +2,44 @@ package remote_web3signer
 
 import (
 	"bytes"
-	"encoding/hex"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"net/url"
-	"strings"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/crypto/bls"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
 )
 
 const (
 	ethApiNamespace = "/api/v1/eth2/sign/"
 	maxTimeout      = 3 * time.Second
 )
 
-// client a wrapper object around web3signer APIs. API docs found here https://consensys.github.io/web3signer/web3signer-eth2.html.
-type client struct {
+// httpSignerClient defines the interface for interacting with a remote web3signer.
+type httpSignerClient interface {
+	Sign(ctx context.Context, pubKey string, request *SignRequest) (bls.Signature, error)
+	GetPublicKeys(ctx context.Context, url string) ([][48]byte, error)
+}
+
+// apiClient a wrapper object around web3signer APIs. API docs found here https://consensys.github.io/web3signer/web3signer-eth2.html.
```

Comment:
Note: unexported methods do not need to follow commenting conventions. Exported ones need to have the name of the thing as the first word, but unexported can be anything you prefer